### PR TITLE
Sandbox directory cleanup

### PR DESCRIPTION
**Pull request by marcmengel**
This addresses  #109.  We now include more files in the transfer_{input,output}_data files so we can just remove
the local sandbox directory after submitting, and again after running condor_transfer_data in jobsub_fetchlog.

The directory is now in $HOME/.cache/jobsub_lite by default.

### CODE REVIEW

**Comment by marcmengel** [comment]
Hm.... dataset dag jobs are going held:
HoldReason = "Error from slot1_63@fnpc7019.fnal.gov: STARTER at 131.225.167.183 failed to send file(s) to <131.225.240.23:9615>: error reading from /storage/local/data1/condor/execute/dir_37014/simple.sh: (errno 2) No such file or directory; SHADOW failed to receive file(s) from <131.225.167.183:46880>"
So not getting wrapper script file back from worker to dagman area?  Need to look into this further...

**Comment by marcmengel** [comment]
Okay, so I was confused by condor renaming the script to condor_exec.exe .  I now have the scripts linking themselves back to their original names.

File: bin/jobsub_submit [hunk 1]
```diff
@@ -235,6 +229,11 @@ def main():
             os.chdir(varg["submitdir"])
             submit(os.path.join(submitdir, "*.cmd"), varg, schedd_name)
 
+    if varg.get("no_submit", False):
+        print(f"Submission files are in: {varg['submitdir']}")
```
Comment (shreyb) [inline bin/jobsub_submit:233]: Love that - very helpful to the user.

File: lib/utils.py [hunk 2]
```diff
@@ -24,9 +24,30 @@
...
+def cleanup(varg: Dict[str, Any]) -> None:
+    """cleanup submit directory etc."""
+    os.chdir(f'{varg["submitdir"]}/..')
+    cleandir(varg["submitdir"])
+    # now clean up old submit directories that weren't
+    # cleaned up by jobsub_submit at the time
+    with os.scandir(".") as it:
+        for entry in it:
+            sb = os.stat(entry.name)
+            if entry.name.startswith("js_") and sb.st_mtime - time.time() > 604800:
```
Comment (shreyb) [inline lib/utils.py:47]: Shouldn't this subtraction be reversed?  That is `time.time() - sb.st_mtime`?

**Review by shreyb — CHANGES_REQUESTED** [review]
All good except for the one comment I had about calculating how old a submit dir is.

File: lib/utils.py (hunk 2 above)
Comment (marcmengel) [reply lib/utils.py:47]: Ack!  yes

Comment (marcmengel) [reply lib/utils.py]: Fixed.  I think I'm getting back all the files I should now for both dag and plain submits, so I think this is ready to merge.

**Review by shreyb — APPROVED** [review]

**Comment by shreyb** [comment]
Just to make sure, are all the changes for this done, or are you still figuring out some of the transfer_{input,output}_files stuff?

**Pull request merged**